spacy/tests/regression/test_issue1-1000.py

```diff
@@ -426,7 +426,7 @@ def test_issue957(en_tokenizer):
 def test_issue999(train_data):
     """Test that adding entities and resuming training works passably OK.
     There are two issues here:
-    1) We have to readd labels. This isn't very nice.
+    1) We have to read labels. This isn't very nice.
```

Comment:
Heh, sorry --- should've punctuated that better.

```suggestion
    1) We have to re-add labels. This isn't very nice.
```